pages/_app.js

```diff
@@ -1,4 +1,5 @@
 import '../styles/globals.css'
+import 'theme';
```

Comment:
Also making a note that this needed to be added as just a `css` file and not a `module.css` file because of the :root issue.